cmd/grr/workflow.go

```diff
@@ -71,6 +72,23 @@ func pullCmd() *cli.Command {
 	}
 	return initialiseCmd(cmd, &opts)
 }
+
+func pullDashboardCmd() *cli.Command {
+	cmd := &cli.Command{
+		Use:   "pull-dashboard <resource-path>",
```

Comment:
Two questions:

- Do we want a completely new command for this?
- Would make sense to extend this new feature to other resources?

I feel like this could be part of one of the existing commands (without adding breaking changes), so I'd like to evaluate that alternative as well.

Reply:
I agree. I mentioned it in the PR description: I'm considering adding a flag to already existing commands instead of adding new commands.

>Would make sense to extend this new feature to other resources?

Maybe. Suggest to discuss it offline together with @malcolmholmes